switchbot_ros/src/switchbot_ros/switchbot_ros_client.py

```diff
@@ -17,6 +17,8 @@ def __init__(self,
                 actionname,
                 SwitchBotCommandAction
                 )
+        rospy.loginfo("Waiting for action server to start.")
+        self.action_client.wait_for_server()
```

Comment:
Basically it is nice to check if action server is active.
But there are some cases that user want to launch this type of client before action server is launched.
And currently this client assumes that initialization of client class has finished even action server is inactive and we want to preserve default behavior of our code so that our robot demonstration works without changing code.

So could you add `timeout` args to `__init__` and set some limited default to it (e.g. '5' or so on) and use it here?

Reply:
I changed timeout in `wait_for_server()` as an additional `__init()__` argment with 5 [sec] default value.

```python
    def __init__(self,
                 actionname='switchbot_ros/switch',
                 topicname='switchbot_ros/devices',
                 timeout=5):
```

```python
        rospy.loginfo("Waiting for action server to start. (timeout: " + str(timeout) + "[sec])")
        self.action_client.wait_for_server(timeout=rospy.Duration(timeout,0))
```